src/webpack-system-register.js

```diff
@@ -114,7 +114,7 @@ WebpackSystemRegister.prototype.apply = function(compiler) {
 		// http://stackoverflow.com/questions/35092183/webpack-plugin-how-can-i-modify-and-re-parse-a-module-after-compilation
 		compilation.plugin('seal', () => {
 			compilation.modules.forEach(module => {
-				let isEntry = module.entry;
+				let isEntry = module.entryModule;
```

Comment:
Could we get this to work with both webpack 1 and webpack 2? Maybe `module.entry || module.entryModule`